jenkins/test-history/buckets.json

```diff
@@ -0,0 +1,11 @@
+{
+    "gs://kubernetes-jenkins/logs/": {
```

Comment:
It's an easy way to make it extensible -- we could add a per-bucket "max job age" this way.